portfolio.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+	<html lang="en-GB">
+	
+	<head>
+		<title>Charlotte Buckley - Portfolio</title>
+		<link rel="stylesheet" type ="text/css" href="style.css"> 
```

Comment:
remember to move your stylesheet below normalize so that you are always overriding the default styles and not the other way around